components/brave_extension/extension/brave_extension/background/reducers/dappDetectionReducer.ts

```diff
@@ -10,14 +10,16 @@ export default function dappDetectionReducer (state = {}, action: Actions) {
     case webNavigationTypes.ON_COMMITTED: {
       if (chrome.braveWallet && action.isMainFrame) {
         chrome.braveWallet.isEnabled((enabled) => {
-          if (enabled) {
-            return
-          }
-          chrome.tabs.executeScript(action.tabId, {
-            file: 'out/content_dapps.bundle.js',
-            allFrames: false,
-            runAt: 'document_start',
-            frameId: 0
+          chrome.braveWallet.isInstalled((installed) => {
```

Comment:
This is just indented 1 with another call. I'm not a huge fan of the nesting, but I wanted to keep the code change small because it's being uplifted.